src/stan_math_backend/Stan_math_code_gen.ml

```diff
@@ -493,10 +495,11 @@ let pp_method_b ppf rt name params intro ?(outro = nop) ?(cv_attr = ["const"])
 (** Print the write_array method of the model class *)
 let pp_write_array ppf {Program.prog_name; generate_quantities; _} =
   pf ppf
-    "template <typename RNG, typename VecR, typename VecI, typename VecVar, \
-     stan::require_vector_like_vt<std::is_floating_point, VecR>* = nullptr, \
-     stan::require_vector_like_vt<std::is_integral, VecI>* = nullptr, \
-     stan::require_std_vector_vt<std::is_floating_point, VecVar>* = nullptr>" ;
+    "template <typename RNG, typename VecR, typename VecI, typename VecVar, @ \
+     \x20 stan::require_vector_like_vt<std::is_floating_point, VecR>* = \
+     nullptr, @ \x20 stan::require_vector_like_vt<std::is_integral, VecI>* = \
+     nullptr, @ \x20 stan::require_std_vector_vt<std::is_floating_point, \
+     VecVar>* = nullptr> @ " ;
```

Comment:
I am not completely sold on this `@ \x20` thing. What you are trying to do is just limit the number of characters in a line right? If so, we can do this with less intervention.

Also dont really like the space before the comma in (see jacobian)
```
template <bool propto__, bool jacobian__ , typename VecR, typename V
```

Reply:
I just thought it was nice looking to have an indentation there but I removed them

Reply:
Oh indentation, I get it. Not an expert on this printing stuff, but I think there is a way to do that without the use of hex. Will give it a shot, else just merge if that is ok with you?

Reply:
Yeah it's fine as is